pkg/results/TablePrinter.go

```diff
@@ -194,16 +194,16 @@ func extract(format Formatter, ips2domains map[string]string, totals Counters, r
 
 // extractTotal is similar to extract but extracts a total from totals rather
 // than an element of an Entry.
-func extractTotal(format Formatter, totals Counters, col OutputColumn) string {
+func extractTotal(format Formatter, totals types.Counters, col OutputColumn) string {
 	switch col {
 	case OutcolInBytes, OutcolBothBytesRcvd:
-		return format.Size(totals.BytesReceived)
+		return format.Size(totals.NBytesRcvd)
 	case OutcolInPkts, OutcolBothPktsRcvd:
-		return format.Count(totals.PacketsReceived)
-	case OutcolOutBytes, OutcolBothBytesSent:
-		return format.Size(totals.BytesSent)
+		return format.Count(totals.NPktsRcvd)
+	case OutcolOutBytes, OutcolBothNBytesSent:
+		return format.Size(totals.NBytesSent)
 	case OutcolOutPkts, OutcolBothPktsSent:
-		return format.Count(totals.PacketsSent)
+		return format.Count(totals.NPktsSent)
```

Comment:
Thanks for switching to `types.Counters`. In a later step, I'll refactor it to have the more verbose, but readable naming `PacketsSent`.

Reply:
You're right. I'v quickly done a find / replace on all of them, should be consistent across the whole package now...